lib/ace/ext/papeeria/tex_completer.coffee

```diff
@@ -314,7 +317,26 @@ define((require, exports, module) ->
           return
 
         if LatexParsingContext.isType(token, "cite")
-          if @citationsUrl? then @citeCache.getReferences(@citationsUrl, callback)
+          if @citationsUrl? then @citeCache.getReferences(@citationsUrl, (err, cites) =>
+            if @filesUrl?
+              # After we get a list of cite autocompletion we want to extract entries from already included biblio files
+              # Let's retrieve a list of files included to the current target from the filesUrl
+              @filesCache.getReferences(@filesUrl, (err, files) =>
+                # Once we have a list of files included to the compilation we can find citation keys that
+                # are (not) included and treat them differently
+                cites.map((cite) =>
+                    if !(cite.meta in files)
```

Comment:
I am curious if it is possible at all. When we build cites.json, we take the list of bib files from aux file produces by tex, which means that all files mentioned in cites belong to the current target by definition. Or do you plan to change this?